document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java

```diff
@@ -43,51 +33,23 @@
     @Mock
     private SmallFullAccountsDataHandler smallFullAccountsDataHandler;
 
-    @Mock
-    private TransactionService transactionService;
-
-    @Mock
-    private Transaction transaction;
-
-    private static final String RESOURCE_ID = "/transactions/091174-913515-326060";
     private static final String RESOURCE_URI_ABRIDGED = "/transactions/091174-913515-326060/accounts/xU-6Vebn7F8AgLwa2QHBUL2yRpk=";
     private static final String RESOURCE_URI_SMALL_FULL = "/transactions/091174-913515-326060/company-accounts/xU-6Vebn7F8AgLwa2QHBUL2yRpk=";
+    private static final String INCORRECT_URI_NO_ACCOUNT = "/incorrect/1234-212/incorrect/1231-1231";
     private static final String REQUEST_ID = "requestId";
 
     @Test
-    @DisplayName("Test DocumentInfoException thrown when error returned from transaction retrieval")
-    void testErrorThrownWhenFailedTransactionRetrieval() throws ServiceException {
-        when(transactionService.getTransaction(anyString(), anyString())).thenThrow(new ServiceException("error"));
-
+    @DisplayName("Test DocumentInfoException thrown when no matching uri is located")
+    void testErrorThrownWhenWhenNoMatchingUriFound() {
```

Comment:
Typo in method name: "testErrorThrownWhenWhenNoMatchingUriFound"

Reply:
updated